deployment/ib-kubernetes-ufm-secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ib-kubernetes-ufm-secret
+  namespace: kube-system
+stringData:
+  UFM_USERNAME: ""
+  UFM_PASSWORD: ""
+  UFM_ADDRESS: ""
+  UFM_HTTP_SCHEMA: ""
+  UFM_PORT: ""
+  UFM_CERTIFICATE: ""
```

Comment:
let set it with default values such as: 
  UFM_USERNAME: "admin"
  UFM_PASSWORD: "12345"